packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -130,3 +130,99 @@ export const glowClass = style([
     zIndex: 0,
   },
 ]);
+
+export const selectContainerClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    backgroundColor: {
+      lightMode: '$gray90',
+      darkMode: '$gray90',
+    },
```

Comment:
nite: If it will stay they same color, you don't need to define lightMode and darkMode separately